perllib/FixMyStreet/SendReport/Email.pm

```diff
@@ -111,4 +111,12 @@ sub send {
     return $result;
 }
 
+sub _get_district_for_contact {
+    my ( $lat, $lon ) = @_;
+    my $district =
+      FixMyStreet::MapIt::call( 'point', "4326/$lon,$lat", type => 'DIS' );
+    ($district) = keys %$district;
+    return $district;
+}
+
```

Comment:
Should this not be in the following commit?

Reply:
Would make more sense, yeah (though I shouldn't have removed it in a978974b5770ebc173d302034cbe179a45cb40d5 and ideally would have force pushed this but bit late now :) ).